pushmi_pullyu.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'daemons', '~> 1.2', '>= 1.2.4'
   spec.add_runtime_dependency 'redis', '~> 3.3'
   spec.add_runtime_dependency 'connection_pool', '~> 2.2'
+  spec.add_runtime_dependency 'openstack', '~> 3.3.10'
```

Comment:
bundler will give us a warning here for dependencies that are too tight. Relax this to simply this:

` spec.add_runtime_dependency 'openstack', '~> 3.3'` 

If you need 3.3.10 or above then do 

spec.add_runtime_dependency 'openstack', '~> 3.3', '>= 3.3.10'